openproject/client.py

```diff
@@ -73,7 +73,7 @@ class WorkPackages(SubClient):
         "lock_version": "lockVersion",
     }
 
-    def _api_payload_from_kwargs(self, **kwargs) -> WorkPackage:
+    def _api_payload_from_kwargs(self, **kwargs: WorkPackage):
```

Comment:
Let's not type hint this for now, as in the future we will support more resources, than just work packages.